redirects.js

```diff
@@ -214,13 +214,6 @@ const redirects = [
     ],
     to: "/articles/testkube-dashboard-explore",
   },
-  {
-    from: [
-      "/guides/going-to-production/authentication/oauth-cli",
-      "/guides/authentication/oauth-cli",
-    ],
-    to: "/articles/oauth-cli",
```

Comment:
should we add a redirect to some other page so any inbound links won't get a 404?

Reply:
Good point, would https://docs.testkube.io/articles/cli be fine?

Reply:
I think so!? should we add a note there regarding this "removal" !?

Reply:
I'm not sure if we should:
* if yes - then maybe better to even keep that page with annotation (but someone may ignore that annotation)
* if not - if someone really needs it, will rather contact us through GitHub, Slack or other way

Reply:
hm.. perhaps we can just move this doc to the legacy-docs section and add the corresponding legacy header?

Reply:
But isn't the Legacy section for things that are deprecated, but still there?

Reply:
true - ok - in that case just remove it and add a redirect to the article you suggested 👍